veil-macros/src/sanitize.rs

```diff
@@ -0,0 +1,58 @@
+trait AttributeFilter {
```

Comment:
I like this implementation, but I feel like this might have some edge cases, so maybe we should write some tests here? If you don't feel like doing it I'd be ok with writing them myself

Reply:
Actually found some bugs thx to this :)